File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/gpt-j/shortest_path_lengths.java
--- 
+++ 
@@ -17,7 +17,7 @@
         for (int i = 0; i < numNodes; i++) {
             for (int j = 0; j < numNodes; j++) {
                 int update_length = Math.min(length_by_path.get(Arrays.asList(i,j)),
-                        length_by_path.get(Arrays.asList(i,k)) + length_by_path.get(Arrays.asList(j,k)));
+                                             length_by_path.get(Arrays.asList(i,k)) + length_by_path.get(Arrays.asList(k,j)));
                 length_by_path.put(Arrays.asList(i,j), update_length);
             }
         }

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/gpt-j/shortest_path_lengths.py
--- 
+++ 
@@ -8,7 +8,7 @@
             for j in range(n):
                 length_by_path[i, j] = min(
                     length_by_path[i, j],
-                    length_by_path[i, k] + length_by_path[j, k]
+                    length_by_path[i, k] + length_by_path[k, j],
                 )
 
     return length_by_path
